platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/merkle/disk/blockstream/BlockStreamConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.state.merkle.disk.blockstream;
+
+import com.hedera.node.config.NetworkProperty;
+import com.hedera.node.config.NodeProperty;
+import com.swirlds.config.api.ConfigData;
+import com.swirlds.config.api.ConfigProperty;
+import com.swirlds.config.api.validation.annotation.Max;
+import com.swirlds.config.api.validation.annotation.Min;
+
+/**
+ * Configuration for block streams. This configuration is enabled when BlockRecordStreamConfig is set to
+ * recordFileVersion=7. It is an error for recordFileVersion != blockVersion.
```

Comment:
This configuration is enabled when BlockRecordStreamConfig is set to recordFileVersion=7.

I do not understand this comment. Where can I see that in the code?